utils/bin/jdtls

```diff
@@ -41,20 +41,20 @@ fi
 # JAR="$HOME/.config/nvim/.language-servers/eclipse.jdt.ls/org.eclipse.jdt.ls.product/target/repository/plugins/org.eclipse.equinox.launcher_*.jar"
 JAR="$HOME/.local/share/nvim/lspinstall/java/plugins/org.eclipse.equinox.launcher_*.jar"
 GRADLE_HOME=$HOME/gradle "$JAVACMD" \
-	-Declipse.application=org.eclipse.jdt.ls.core.id1 \
-	-Dosgi.bundles.defaultStartLevel=4 \
-	-Declipse.product=org.eclipse.jdt.ls.core.product \
-	-Dlog.protocol=true \
-	-Dlog.level=ALL \
-	-javaagent:$HOME/.local/share/nvim/lspinstall/java/lombok.jar \
-	-Xms1g \
-	-Xmx2G \
-	-jar $(echo "$JAR") \
```

Comment:
@ChristianChiarulli Tagging you because I know I'm guilty of having done the same change in the past, and it seems that it breaks things for you.
Does it need to expand into multiple strings ?

Reply:
I'm not exactly sure what caused the break, but until we can confirm this doesn't break things in the future I think we should exclude this file somehow.

Reply:
is this okay now? I've added a directive ` # shellcheck disable=SC2116`. And fixed the other ones, since they were pretty straight-forward.